### PR TITLE
fix: remove empty search result from suggestions (history)

### LINKED ISSUE #4308: Remove empty search from history
Probably we don't need to show an empty past search.

image

on white mode aswell: 
image
 but its invisible

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #4308
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [x] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/search/SearchSuggestion.vue [hunk 1]
```diff
@@ -452,6 +452,9 @@ export default class SearchSuggestion extends mixins(PrefixMixin) {
     }
 
     return this.searched.filter((option) => {
+      if (option.name.trim().length === 0) {
```
Comment (roiLeo) [inline components/search/SearchSuggestion.vue:455]:
```suggestion
      if (!option.name.trim().length) {
```

**Review by roiLeo — COMMENTED** [review]
Search history is uppercase in navbar but not in landing searchbar

Screenshot 2022-11-13 at 18-49-51 KodaDot - Kusama NFT Market Explorer
Screenshot 2022-11-13 at 18-49-33 KodaDot - Kusama NFT Market Explorer

File: components/search/SearchSuggestion.vue (hunk 1 above)
Comment (petersopko) [reply components/search/SearchSuggestion.vue]: 👍🏻

**Comment by petersopko** [comment]
> Search history is uppercase in navbar but not in landing searchbar
> 
> Screenshot 2022-11-13 at 18-49-51 KodaDot - Kusama NFT Market Explorer Screenshot 2022-11-13 at 18-49-33 KodaDot - Kusama NFT Market Explorer

https://github.com/kodadot/nft-gallery/issues/4328

**Review by roiLeo — APPROVED** [review]
✅

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 25.66 USD/KSM ~ 1.169 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x41d22f91878c4b222b6f5d420d98d011114e9e9f114e5eae63aa40af9c368ac4](https://kusama.subscan.io/extrinsic/0x41d22f91878c4b222b6f5d420d98d011114e9e9f114e5eae63aa40af9c368ac4)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
